horace_core/sqw/@sqw/shift_pixels.m

```diff
@@ -103,7 +103,8 @@
         data.npix = data.pix.num_pixels;
         eps_lo = min(data.pix.dE);
         eps_hi = max(data.pix.dE);
-        data.urange(:,4) = [eps_lo;eps_hi];
+        % TODO: meaning?
```

Comment:
?

Reply:
Its similar to `section`  but here is also not clear why one wants run recompute anyway. I do not believe tthere is unit test to check this so Needs careful investigation